handler.go

```diff
@@ -156,7 +164,7 @@ func appendPair(pairs []any, groupPrefix string, attr slog.Attr) []any {
 			key = groupPrefix + "." + key
 		}
 
-		pairs = append(pairs, key, attr.Value)
+		pairs = append(pairs, key, attr.Value.Resolve())
```

Comment:
Calling Resolve() on Value at logging-time is the only significant change to the implementation.